entrypoint.sh

```diff
@@ -30,4 +30,4 @@ mklink outputs
 import outputs
 
 # Start application
-python launch.py $*
+python entry_with_update.py $*
```

Comment:
users normally do not expect Fooocus to be suddenly updated when running a container tagged with a specific version

```suggestion
python launch.py $*
```